### PR TITLE
Fix code typos

### PR DESCRIPTION
**Pull request by matheus-consoli**
The spiritual successor of #136

### CODE REVIEW

File: components/dada-execute/src/heap_graph.rs [hunk 1]
```diff
@@ -51,7 +51,7 @@ tables! {
     pub(crate) struct Tables {
         stack_frame_nodes: alloc StackFrameNode => StackFrameNodeData,
         objects: alloc ObjectNode => ObjectNodeData,
-        datas: alloc DataNode => DataNodeData,
```
Comment (XFFXFF) [inline components/dada-execute/src/heap_graph.rs:54]: Not sure if this is a typo

Comment (matheus-consoli) [reply components/dada-execute/src/heap_graph.rs:54]: yeah, I also was in doubt about this one... data is already in plural form, from my understanding, and `datas` don't really is a word. Even so, I have done and undone this specific some times.
If it's a design choice, let me know and I change it back.

Comment (brson) [reply components/dada-execute/src/heap_graph.rs:54]: Yeah, I think this is an intentional misspelling for naming consistency. I think it's best to leave it.

**Comment by brson** [comment]
Thanks for teh spell checking!

File: components/dada-execute/src/heap_graph.rs (hunk 1 above)
Comment (matheus-consoli) [reply components/dada-execute/src/heap_graph.rs]: Makes sense to me! Removed it!

**Comment by XFFXFF** [comment]
Thanks!  
bors r+

**Pull request merged**